css/idcard.css

```
@@ -434,3 +434,7 @@
 #biliscope-id-card #word-cloud-toggler svg {
     padding: 2px 0px;
 }
+
+#biliscope-id-card #word-cloud-toggler .arrow-up, .arrow-down {
```

```suggestion
#biliscope-id-card .arrow-up,
#biliscope-id-card .arrow-down {
```

应该是这样的吧。你现在的`.arrow-down`应该是不受前面的部分制约的。

---

我的我的，太久没写 css 忘记 css 选择器的写法了。把 lint 关了一下子没想起来。